tzwhere/tzwhere.py

```diff
@@ -26,9 +26,11 @@ class are instantiated and queried directly
     import numpy
     WRAP = numpy.asarray
```

Comment:
I think this would be cleaner if you did:

WRAP = functools.partial(numpy.asarray, dtype=object)

Then WRAP will always get the dtype=object argument, and you won't have to add or branch on`numpy_avail`.

Although you will have to import [functools](https://docs.python.org/3/library/functools.html#functools.partial). :-)